test/e2e/multicast.go

```diff
@@ -202,19 +202,36 @@ func testMulticastIGMPQuery(f *framework.Framework, clientNodeInfo, serverNodeIn
 		framework.Failf("failed to retrieve multicast IGMP query: " + err.Error())
 	}
 
-	// The multicast listener pod join multicast group (-B 224.1.1.1), UDP (-u), during (-t 30) seconds, report every (-i 5) seconds
-	ginkgo.By("multicast listener pod join multicast group")
-	e2ekubectl.RunKubectl(f.Namespace.Name, "exec", multicastListenerPod, "--", "/bin/bash", "-c", fmt.Sprintf("iperf -s -B %s -u -t 30 -i 5", mcastGroup))
+	ginkgo.By("multicast listener pod joins multicast group and verifies IGMP query")
+	maxDuration := 2 * time.Minute
+	start := time.Now()
```

Comment:
If you use Eventually you don't need these.

Reply:
Valid point, @qinqon. Let me do it using `eventually`.